card/card.gemspec

```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |s|
     ["uuid",                       "~> 2.3"],
     # with carrierwave 0.11.1 and 0.11.2 wagn hangs in a loop on creating
     # style and script machine output
-    ["carrierwave",                "<= 0.11.0"],
+    #["carrierwave",                "<= 0.11.0"],
```

Comment:
so, presumably development is still active and we'll be able to go back to the gem when it comes out?